benchmarks/build.gradle

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'java-library'
+    id "me.champeau.jmh" version "0.6.8"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    api project(':core')
```

Comment:
use implementation?
https://stackoverflow.com/questions/44413952/gradle-implementation-vs-api-configuration

Reply:
Fixed in https://github.com/opensearch-project/sql/pull/1278/commits/35af12647bdfbe79ea000b815176a69a9a013ab9. Thanks!